modular-docs-manual/content/topics/module_creating-modules.adoc

```diff
@@ -10,6 +10,8 @@ After identifying and creating the assemblies, each assembly should identify the
 +
 Each assembly should already have the names of the modules that should be included in the assembly. Now you just need to create the actual files for those modules.
 
+Be sure to follow the conventions for xref:anchor-and-file-names[naming anchors and files]. For example: `guided-decision-tables-con.adoc`
```

Comment:
The example is wrong here, should be `con_guided-decision-tables.adoc`. I've probably missed this before. Otherwise LGTM.

Reply:
Great catch, @tradej! Fixed. Thanks.